tests/console/mta.pm

```diff
@@ -37,7 +38,7 @@ sub run {
     # test email transmission
     assert_script_run 'echo "FOOBAR123" | mail root';
     assert_script_run 'postqueue -p';
-    script_run 'cat /var/mail/root';
+    sleep(1);
```

Comment:
This is due to postfix not processing/delivering the mail fast enough right?

Reply:
Yes, try to run this command anywhere, I could see on every machine the first mail in queue.

```
# echo "FOOBAR123" | mail root && postqueue -p && grep FOOBAR123 /var/mail/root
Mail queue is empty
FOOBAR123
scruffy:~ # echo "FOOBAR123" | mail root && postqueue -p && grep FOOBAR123 /var/mail/root
Mail queue is empty
FOOBAR123
FOOBAR123
scruffy:~ # echo "FOOBAR123" | mail root && postqueue -p && grep FOOBAR123 /var/mail/root
Mail queue is empty
FOOBAR123
```

Reply:
This should be fixed in a better way then and not as part of this commit IMO.

Reply:
Can't wait for the improvement on this puzzling issue.
I see only simple and short wait for mail queue to get processed in super fast test.

Reply:
It could be changed to `wait_serial` on `tail -f` and send `ctrl-c` afterwards for instance.

Reply:
good enough ?

Reply:
I was actually fine with the first version.

A decent fix would be to check the postfix queue until the email is actually delivered (waitserial on the journal?) and then check the mail queue... but this is outside of the scope of this PR FWIW

Reply:
that's what `assert_script_run 'until postqueue -p|grep "Mail queue is empty";do sleep 1;done';` does or ?

Reply:
> that's what `assert_script_run 'until postqueue -p|grep "Mail queue is empty";do sleep 1;done';` does or ?

Yeah :)